src/elements/Image/Image.tsx

```diff
@@ -43,7 +43,7 @@ export function Image({ children, className, node, onDownload, onPreviewOpen, ..
     const containerStyle = getContainerStyle(node);
     const handleRolloverClick = () => setIsPreviewOpen(true);
     const handleImagePreviewClose = () => setIsPreviewOpen(false);
-    const title = stringifyNode(node);
+    const title = stringifyNode(node).trim();
```

Comment:
This also fixes a bug where the tooltip would appear empty:

![Screenshot 2023-06-06 at 17 28 32](https://github.com/prezly/content-renderer-react-js/assets/4209081/e7ec359c-b647-4324-8722-e3b13adc9061)